docs/site/Integrate-with-infrastructure.md

```diff
@@ -0,0 +1,10 @@
+---
+lang: en
+title: 'Integrate with Other Infrastructure'
```

Comment:
`Infrastructures`?

Reply:
Fixed. Actually on a  previous comment from @raymondfeng , it should be `Integrate with Infrastructures`. :) Thanks.